dev-libs/bglibs/files/bglibs-2.04-prefix.patch

```diff
@@ -0,0 +1,11 @@
+--- a/Makefile
++++ b/Makefile
+@@ -863,7 +863,7 @@ path/mktemp.lo path/mktemp.o: ltcompile path/mktemp.c systime.h include/bglibs/p
+ 	./ltcompile path/mktemp.c
+ 
+ perl-head.pl: 
+-	( set -e; PATH="/bin:/usr/bin:/usr/local/bin:$$PATH"; export PATH; \
++	( set -e; \
+ 	  perl=`which perl`; \
```

Comment:
While touching it, you may want to fix the `which` use too..

Reply:
sys-apps/which is already in BDEPEND.

Reply:
Well, yes, the point is we're trying to purge `which`, so if we see a use of it that's trivial, we should try patch it out.

Reply:
I think done.

Is calling host's /usr/bin/env allowed on prefix?

Reply:
Yeah, I think it's OK (or I hope so)